doc/CRD-generator.md

```diff
@@ -26,7 +26,215 @@ dependencies {
 Now you can define a `class` that extends `io.fabric8.kubernetes.client.CustomResource`
 and the corresponding CRD is generated in the folder: `target/classes/META-INF/fabric8`
 
-### Supported annotations
+For example, code similar to:
+
+```java
+@Group("org.example")
+@Version("v1alpha1")
+@ShortNames("ex")
+@Plural("examples")
```

Comment:
Note that you only need to specify the plural if you're not satisfied with the one generated by default, which in this case would be `examples` already… :)